src/commands/review.js

```diff
@@ -63,16 +72,81 @@ mutation($projectName: String!, $responses: [CLINamedSurveyResponse]!) {
     .then(data => data.saveProjectReviewCLISurveyResponses)
 }
 
-function projectReviewRecordedSuccessMessage(projectName, args) {
+function projectReviewRecordedSuccessMessage(projectName, args, completed) {
+  let msg = ''
   if (args.completeness && args.quality) {
-    return `Completeness and quality scores captured for #${projectName}!`
+    msg += `Completeness and quality scores captured for #${projectName}!`
+  } else if (args.completeness) {
+    msg += `Completeness score captured for #${projectName}!`
+  } else if (args.quality) {
+    msg += `Quality score captured for #${projectName}!`
   }
 
-  if (args.completeness) {
-    return `Completeness score captured for #${projectName}!`
+  if (completed) {
+    msg += 'Review is complete. Thank you for your input.'
+  } else {
+    msg += 'Review is not yet complete.'
```

Comment:
Do we need whitespace between these two parts of `msg`? It seems like we could end up with:

`Quality score captured for #red-bull!Review is not yet complete.`

Am I reading it wrong?